modules/olm-mirroring-catalog-airgapped.adoc

```diff
@@ -32,7 +32,7 @@ $ oc adm catalog mirror \
 <2> Specify the content to mirror to local files in your current directory.
 <3> Optional: If required, specify the location of your registry credentials file.
 <4> Optional: If you do not want to configure trust for the target registry, add the `--insecure` flag.
-<5> Optional: Specify which platform and architecture of the index image to select when multiple variants are available. Images are specified as `'<platform>/<arch>[/<variant>]'`. This does not apply to images referenced by the index. Valid values are `linux/amd64`, `linux/ppc64le`, `linux/s390x`, and `.*`
+<5> Optional: Specify which platform and architecture of the index image to select when multiple variants are available. Images are specified as `'<platform>/<arch>[/<variant>]'`. This does not apply to images referenced by the index. Valid values are `linux/amd64`, `linux/ppc64le`, `linux/s390x`, `linux/aarch64`, and `.*`
 +
```

Comment:
And this one should `linux/arm64`, I run command using `linux/aarch64` will hit error
```
QE_index_image="registry.redhat.io/redhat/redhat-operator-index:v4.11"
oc adm catalog mirror --index-filter-by-os='linux/aarch64' ${QE_index_image} quay.io/lwan0/test --manifests-only
error: the image is a manifest list and contains multiple images - use --index-filter-by-os to select from:

  OS            DIGEST
  linux/amd64   sha256:7e322e0b058b53b1014e7c4318df3c195afbf9e5875c75335d561200907709b6
  linux/arm64   sha256:7fecde7bb8f0f5c3fa558c2272bb21dce118dde1e85841dd73d37a7cfaa22e62
  linux/ppc64le sha256:a0c861166544f225065a3c50bead54a4f4b2d76d2e337b6949ebec03986f1195
  linux/s390x   sha256:f5fd090ef8b96a25ac8c526fa8ea0e8405e5e0dfc65314bbb9fb9edb2625e7a5
```